DSharpPlus/Entities/Interaction/DiscordInteraction.cs

```diff
@@ -102,6 +102,18 @@ public DiscordChannel Channel
         [JsonProperty("message")]
         internal DiscordMessage Message { get; set; }
 
+        /// <summary>
+        /// Gets the selected language of the invoking user.
```

Comment:
This should be along the lines of "Gets the locale of the invoking user." to be consistent :)

Reply:
Fixed.